lib/galaxy/jobs/runners/pulsar.py

```diff
@@ -205,6 +206,14 @@ def __init_pulsar_app( self, pulsar_conf_path ):
             log.info("Loading Pulsar app configuration from %s" % pulsar_conf_path)
             with open(pulsar_conf_path, "r") as f:
                 conf.update(yaml.load(f) or {})
+        if "job_metrics_config_file" not in conf:
+            conf["job_metrics"] = self.app.job_metrics
```

Comment:
``job_metrics`` is intentional here. If the user doesn't specify an explicit configuration file for Pulsar that is different - I just pass through the actual object (``job_metrics``) through to Pulsar and it re-uses it - instead of constructing a new one from a configuration file.